src/Commands/InitCommand.php

```diff
@@ -388,29 +401,31 @@ protected function publishMigration(View $view, string $migrationName): void
         file_put_contents("database/migrations/{$migrationName}", "<?php\n\n{$data}");
     }
 
-    protected function createOrUpdateConfigFile(string $fileName, string $separator, array $data): void
+    protected function createOrUpdateConfigFile(string $separator, array $data, ...$fileNames): void
```

Comment:
it's not good to change this function as it will works only in case we'll need to update several files with the same content. I not sure it rather often case.

Let's just call original function twice if it needs OR create a wrapper `updateConfigFiles`